Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:14.2 AS build-fe
+WORKDIR /src/
+COPY ui  .
+RUN yarn
+RUN yarn build
+
+FROM golang:1.14-alpine AS build-be
+WORKDIR /src/
+COPY . .
```

Comment:
same tip here, we can copy `go.mod` and `go.sum` first, run `go mod download`, then copy the source and run `go build`